src/main/java/com/alex/eshop/mapper/CategoryCreateMapper.java

```diff
@@ -0,0 +1,11 @@
+package com.alex.eshop.mapper;
+
+import com.alex.eshop.dto.CategoryCreateDTO;
+import com.alex.eshop.entity.Category;
+import org.mapstruct.Mapper;
+
+@Mapper
+public interface CategoryCreateMapper {
```

Comment:
There is no reason to create Mapper for each DTO type. Use one mapper `CategoryMapper` for all `CategoryEntity-CategoryAnyDto` mapping.